### PR TITLE
Refactor directional images

### PR DESCRIPTION
**Pull request by Sindex42**

### CODE REVIEW

**Review by CKKH — APPROVED** [review]
Another lovely refactor.

**Review by matharotheelf — APPROVED** [review]
Awesome refactoring

File: src/map.txt [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-1000111001110001
-10001E0000E10001
-1000000000000001
-10000000H0000001
-10001E0000E10001
-1000111001110001
-1000000000000001
-1E000000000000E1
-1111111111111111
+11111111111111111111111111111111
```
Comment (michaelnguyen974) [inline src/map.txt:1]: nice map extension

File: src/hero.py [hunk 2]
```diff
@@ -71,57 +74,12 @@ def end_game(self):
...
-    def load_left_image(self):
-        ''' Loads leftward facing sprites '''
-
-        for image in os.listdir('images/orthogonal_boy/orthogonal_boy_left'):
-            path = os.path.join(
-                'images/orthogonal_boy/orthogonal_boy_left', image)
-            self.left_images.append(
-                pg.transform.scale(
-                    pg.image.load(path), (TILESIZE - 1, TILESIZE - 1)))
+        ''' Assigns directional images to appropriate lists '''
```
Comment (michaelnguyen974) [inline src/hero.py:77]: excellent refactor

**Review by michaelnguyen974 — APPROVED** [review]

**Pull request merged**